### ISSUE TITLE
Reinsert MightyGau-8Avenger 

### ISSUE THREAD

**Issue by scooby78** [scooby78/160thSOR#72]
Recent fixes for the A-10 gun look to be completed and ready for reinsertion.

https://github.com/ZabuzaW/MightyGau-8Avenger/releases

**Comment by yettie79** [comment]
Ill say to you what ill say to everyone else, is it needed or wanted? Is it a mod that you would use in most missions ? Is not having this mod stopping you from doing anything? Has this mod been tested as per testing guide? Does the community want it? And where does the white go when the snow melts?

**Comment by scooby78** [comment]
Lol... well....

It was only removed as a result of the Jets dlc and then an rhs incompatibility (also from jets dlc). 

So this is less about adding a new mod and more about returning an existing mod back into the pack..

The current gun on the a-10 makes it useless against targets that the a-10 is designed for..  so with this mod off there is less fun runs and more missile runs..  ironically I think this is more for inf than anything.  Plus the ‘Brrrrrrrrrttt’ sound will be back..

FYI this mod has been used for the better part of two years prior to it being removed for incompatibility reasons only..

**Comment by Diesel486** [comment]
Still in need of testing? I can set something up this evening if need. Either local Host or if someone can do server side stuff then can run a server test

**Comment by scooby78** [comment]
Yeah I’ll need to run everything through it’s paces as a whole (all mod updates loaded) and make sure there isn’t any funny shit happening before the final pack update..

**Comment by scooby78** [comment]
Bullet hit sounds seemed to have and "unusual echo".

Further testing required without the performance pbo

**Comment by yettie79** [comment]
Tested with 1 INF person on the ground. 
we did 4 attack runs with no issue found.

**Comment by Deek356** [comment]
Deek to run another test 
Aviation do a fly by and shoot with infantry on the ground listening.

**Comment by Diesel486** [comment]
test run. 3 players, did gun runs and different distances, altitudes and behind mountains from the ground-based player. all audible seems normal. all particle effects seem normal though splash damage seems a bit low. possible error in rpt file "Server: Object 7:295 not found (message Type_114)" were many of these with different message type's, needs further testing on another pc or server to check for this
